app/assets/sass/application.scss

```diff
@@ -1,21 +1 @@
-// global styles for <a> and <p> tags
-$govuk-global-styles: true;
-$govuk-new-link-styles: true;
-
-// We can't mount GOV.UK Frontend's assets at root as it's done automatically by the extensions framework.
-$govuk-assets-path: '/govuk/assets/';
-
-// Import GOV.UK Frontend and any extension styles if extensions have been configured
-@import ".tmp/sass/extensions";
-
-// Patterns that aren't in Frontend
-@import "patterns/contents-list";
-@import "patterns/mainstream-guide";
-@import "patterns/pagination";
-@import "patterns/related-items";
-@import "patterns/step-by-step-header";
-@import "patterns/step-by-step-nav";
-@import "patterns/step-by-step-related";
-@import "patterns/task-list";
-
 // Add extra styles here, or re-organise the Sass files in whichever way makes most sense to you
```

Comment:
@NoraGDS we don't really want people to re-organise sass however they want, so what do you think of making this just 
```
// Add extra styles here
```

Reply:
yes, agree with keeping it simple. my only question is will users know we're referring to our templates with 'styles' here?

Reply:
@joelanman I can force the comment line to be replaced with the simpler comment. @NoraGDS I'm not sure what you mean here?  All the code from our sass templates (I might have the wrong end of the stick) has been moved to lib and only the custom sass has been left behind whether they be custom sass in the applications.scss file or custom scss files and customer pattern scss files.

Reply:
@NoraGDS by 'styles' we mean any custom scss (sass) or css of their own they want to add to their prototype

Reply:
ok. thanks.

Reply:
Comment line is now updated to:
`// Add extra styles here`